playground/worker/__tests__/es/es-worker.spec.ts

```diff
@@ -57,7 +57,7 @@ describe.runIf(isBuild)('build', () => {
   test('inlined code generation', async () => {
     const assetsDir = path.resolve(testDir, 'dist/es/assets')
     const files = fs.readdirSync(assetsDir)
-    expect(files.length).toBe(27)
+    expect(files.length).toBe(28)
```

Comment:
I'm not sure why the file count changes.

Reply:
May be delete `playground/worker/modules/test-plugin.tsx`